src/main/java/org/spongepowered/api/text/sink/package-info.java

```diff
@@ -22,7 +22,4 @@
  * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
```

Comment:
This package-info (api/text/sink/package-info.java) doesn't exist anymore, where did it come from? In any case, shouldn't be making any changes